libcaf_net/caf/net/socket.hpp

```diff
@@ -55,6 +55,10 @@ void close(socket x);
 /// @relates socket
 std::errc last_socket_error();
 
+/// Returns the string representation of a given socket error code.
+/// @relates socket
+std::string socket_error_as_string(std::errc err);
```

Comment:
This is never used and also lacks an implementation on Windows. Why did you add it?